IUFRO-HQ.csl

```diff
@@ -0,0 +1,128 @@
+  <info>
+    <title>International Union of Forest Research Organizations - Headquarters</title>
+    <title-short>IUFRO - HQ</title-short>
+    <id>http://www.zotero.org/styles/international-union-of-forest-research-organizations-headquarters</id>
+    <link href="http://www.zotero.org/styles/international-union-of-forest-research-organizations-headquarters" rel="self"/>
+    <author>
+      <name>Nelson Grima</name>
+      <email>grima@iufro.org</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <updated>2020-11-23T07:51:25+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" et-al-min="3"/>
+    </names>
+  </macro>
+  <macro name="author-count">
+    <names variable="author">
+      <name form="count"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="issued">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher"/>
+      <text variable="publisher-place" suffix="."/>
+    </group>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=".">
+      <label form="short" text-case="capitalize-first" prefix="(" suffix=") "/>
+      <name and="text" et-al-min="5" initialize-with="."/>
+    </names>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
+    <sort>
+      <key macro="author-short" names-min="1" names-use-first="1"/>
+      <key macro="author-count" names-min="3" names-use-first="3"/>
+      <key macro="author" names-min="3" names-use-first="1"/>
+      <key macro="issued"/>
+      <key variable="title"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <text macro="issued"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="11" et-al-use-first="10">
+    <sort>
+      <key macro="author-short" names-min="1" names-use-first="1"/>
+      <key macro="author-count" names-min="3" names-use-first="3"/>
+      <key macro="author" names-min="3" names-use-first="1"/>
+      <key macro="issued"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=" ">
+        <text macro="author"/>
+        <text macro="issued" prefix="(" suffix=")."/>
+        <choose>
+          <if type="article article-magazine article-newspaper article-journal review manuscript paper-conference" match="any">
+            <text variable="title" text-case="sentence" font-style="italic" suffix="."/>
```

Comment:
We don't use `text-case="sentence"` for titles and it will be deprecated in the future (as it can't handle proper names). If you want titles in sentence case, just don't use a `text-case` attribute. You should assume titles are stored in sentence case in the metadata.

Reply:
Thank you, I was not aware of this issue. I removed the 'text-case' attribute in the new version.